row_023_app/row_023_app_safe.py

```
@@ -0,0 +1,119 @@
+        return render_template('index.html', invalidImage=True, init=True)
+    ## logic to call the service
+    t = read_tensor_from_image_file(
+      filename,
+      input_height=input_height,
+      input_width=input_width,
+# {/fact}
+      input_mean=input_mean,
+      input_std=input_std)
+    ##
+    results = sess.run(output_operation.outputs[0], {input_operation.outputs[0]:t})
+    results = np.squeeze(results)
```

Ruff (0.14.2)

`np` may be undefined, or defined from star imports

(F405)